### DISCUSSION TITLE
这种大量条纹怎么处理？

### DISCUSSION THREAD

**Discussion by yuygfgg** [vcb-s/guides#17] [General]
<img width="1920" alt="截屏2024-08-29 19 35 52" src="https://github.com/user-attachments/assets/26a0f56b-2c95-4c26-80cc-7449dafcc374">
<img width="1920" alt="截屏2024-08-29 19 33 55" src="https://github.com/user-attachments/assets/82a0b6c6-9550-4e6a-8e12-16eb6b670f98">

**Comment by dtlnor** [comment]
這種條紋是紋理吧。反倒是色度質量堪憂，這是BDMV嗎？

**Reply by yuygfgg** [reply]
webrip

义妹生活E08

**Reply by dtlnor** [reply]
web這個品質就合理了。條紋他前景（人物）沒有，背景才有，判斷爲特效。
